hibernate/common/src/main/java/org/n52/sos/ds/hibernate/dao/series/SeriesObservationDAO.java

```diff
@@ -732,7 +732,7 @@ protected void addObservationIdentifiersToObservation(ObservationIdentifiers obs
                                 observationIdentifiers.getObservableProperty(), observationIdentifiers.getProcedure(),
                                 session);
         ((SeriesObservation) hObservation).setSeries(series);
-        seriesDAO.updateSeriesWithFirstLatestValues(series, hObservation, session);
+//        seriesDAO.updateSeriesWithFirstLatestValues(series, hObservation, session);
```

Comment:
Lets document this, but not implement it yet.  I'd like to minimize changes to 52n as much as possible.